sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -888,6 +888,175 @@ case class MapFromEntries(child: Expression)
     copy(child = newChild)
 }
 
+@ExpressionDescription(
+  usage = """
+    _FUNC_(map[, ascendingOrder]) - Sorts the input map in ascending or descending order
```

Comment:
Could you leave a high-level description of the expression here, and add `arguments` w/ detailed description of `map` and `ascendingOrder`.

Since the expression focuses on sorting, describe characteristics of the sort alg: stable or not, and etc.